dubbo-cluster/src/main/java/org/apache/dubbo/rpc/cluster/support/registry/ZoneAwareClusterInvoker.java

```diff
@@ -92,8 +92,8 @@ public Result doInvoke(Invocation invocation, final List<Invoker<T>> invokers, L
         }
 
 
-        // load balance among all registries, with registry weight count in.
-        Invoker<T> balancedInvoker = select(loadbalance, invocation, invokers, null);
+        // Use random load balance among all registries, with registry weight count in.
+        Invoker<T> balancedInvoker = select(ExtensionLoader.getExtensionLoader(LoadBalance.class).getExtension("random"), invocation, invokers, null);
```

Comment:
hi, @zhangyz-hd ,is it properly to use random here

Reply:
I think "random" and "roundrobin" are both suitable, but not the default SPI. So extract it as a parameter and give it to the user to configure it?

Reply:
> extract it as a parameter and give it to the user to configure it
Agree, I think this would be more suitable